src/Raven.Server/Raven.Server.csproj

```diff
@@ -152,10 +152,15 @@
   </ItemGroup>
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="AWSSDK.Core" Version="3.7.302.16" />
+    <PackageReference Include="AWSSDK.Glacier" Version="3.7.300.55" />
+    <PackageReference Include="AWSSDK.S3" Version="3.7.305.31" />    
+    <PackageReference Include="Azure.Identity" Version="1.10.4" />
```

Comment:
Please remove that

Reply:
I meant `AWSSDK.*` - because it's duplicated

Reply:
done